src/main/kotlin/com/linchpino/ai/model/Prompt.kt

```diff
@@ -1,8 +1,15 @@
-package com.linchpino.ai.model;
+package com.linchpino.ai.model
 
-public class Prompt {
+class Prompt(private val requestDetail: RequestDetail) {
+    override fun toString(): String {
+        return ROADMAP_PROMPT
+            .replace("_targetLevel_", requestDetail.targetLevel)
+            .replace("_summary_", requestDetail.resume.summary)
+            .replace("_experience_", requestDetail.resume.experience)
+    }
 
-    public static final String ROADMAP_PROMPT = """
+    companion object {
+        val ROADMAP_PROMPT: String = """
```

Comment:
there is no need for a companion object here. define this string as instance member. if you want you can use top level members too.
const val Roadmap_Prompt = "example"